package.json

```diff
@@ -51,6 +51,7 @@
     "hmpo-logger": "6.1.1",
     "i18next": "^21.9.2",
     "i18next-fs-backend": "^1.1.5",
+    "lodash": "^4.17.21",
```

Comment:
are we able to import just `lodash.fromparis`

Like `lodash.differencewith` below